mistune/plugins/tabulate.py

```diff
@@ -0,0 +1,1772 @@
+# -*- coding: utf-8 -*-
```

Comment:
I assume we need to include the *tabulate* license if we bundle this module?

Reply:
I added the license doc as docstring in tabulate.py, is this the proper way to include it?

Reply:
That's probably sufficient.